packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
+      Before you start the node, you should review the configuration options and the default values to determine whether you want to make any changes to the configuration of the node.
+      For information about this next step, see [Review the default configuration](#review-the-default-configuration).
+
+## Build from source
+
+In most cases, you should run Chainweb nodes using officially released `chainweb-node` binaries or from the binary packaged in officially released Docker images that you can download from the [Releases](https://github.com/kadena-io/chainweb-node/releases) page. 
+However, if you choose to build the chainweb-node binary yourself, you should first ensure that you have an officially released and tagged version of the source code. 
+Tagged versions of the source code are tested extensively to ensure that they are compatible with all nodes in the Chainweb network.
+
+You shouldn't build `chainweb-node` from the `master` branch if you plan to run the node as part of a Kadena public network.
+
+### Download source code
+
+To download tagged source code:
+
+1. Open the [Releases](https://github.com/kadena-io/chainweb-node/releases) page.
+
+2. Download the compressed archive for your working environment:
+
+   ![Download source code](/assets/docs/chainweb-node-assets.png)
+
+3. Unzip and extract the compressed archive by double-clicking or by running a command similar to the following:
+
+   ```bash
+   tar -xvzf chainweb-node-2.24.1.tar.gz
+   ```
+
+1. Change to the `chainweb-node` source code directory. 
+
+After you have downloaded and unpacked the source code, you have two options for building the `chainweb-node` binary from the source:
+
+- You can build the binary using the native Haskell toolchain.
+- You can build the binary using the Nix package manager.
+
+### Build with Haskell
+
+To build with the native Haskell toolchain:
+
+1. Download and install the following Haskell tools:
+   
+   - [Glasgow Haskell Compiler (GHC)](https://www.haskell.org/ghc/), `ghc-9.6.5`, or later.
+   - [Haskell build tool CABAL](https://www.haskell.org/cabal/), `cabal`, version 3.4, or later.
+
+   You can download, install and manage the Haskell toolchain using `ghcup`.
+   To install on Linux, macOS, FreeBSD, or WSL2, go to [GCHCup](https://www.haskell.org/ghcup/) to download the main Haskell installer, then follow the installation instructions to install the toolchain.
+
+1. Install the development versions of the following required libraries: 
+
+   - gflags-dev 
+   - snappy-dev 
+   - zlib-dev
+   - lz4-dev
+   - bz2-dev
+   - zstd-dev
+
+   If the host uses the advance package tool (apt), you can install these libraries by running the following command:
+   
+   ```bash
+   apt-get install ca-certificates libssl-dev libgmp-dev libsnappy-dev zlib1g-dev liblz4-dev libbz2-dev libgflags-dev libzstd-dev
+   ```
+
+   On macOS, you can install these libraries by running the following command:
```

Comment:
`brew` will also need to be installed, as well as xcode-tools, which have to be installed separately.